### DISCUSSION TITLE
Any chance this can support displaying face styles vs just face structure?

### DISCUSSION THREAD

**Discussion by markbland82** [hacksider/Deep-Live-Cam#723] [Bounty Ideas]
Would love to see this support whatever colours are in the image (any colour vs just skin tones) so as to keep the styling of the face vs just the structure.

**Comment by KRSHH** [comment]
That would require a different model. we don't train our own weights for Deep-Live-Cam, If you come across that type of open source faceswap model you can inform us.

**Reply by markbland82** [reply]
Thanks!
